src/Apps/Partner/Components/PartnerArtists/PartnerArtistList/PartnerArtistList.tsx

```diff
@@ -1,95 +1,150 @@
 import * as React from "react"
-import { PartnerArtistList_artists$data } from "__generated__/PartnerArtistList_artists.graphql"
+import { PartnerArtistList_partner$data } from "__generated__/PartnerArtistList_partner.graphql"
 import { createFragmentContainer, graphql } from "react-relay"
-import { Box, Column, GridColumns, Text } from "@artsy/palette"
-import { groupArtists } from "Apps/Partner/Components/PartnerArtists/partnerArtistsUtils"
-import { ColumnSpan } from "@artsy/palette/dist/elements/GridColumns/calculateGridColumn"
-import { Media } from "Utils/Responsive"
-import { PartnerArtistItemFragmentContainer as PartnerArtistItem } from "./PartnerArtistItem"
-import { Carousel } from "Components/Carousel"
-import { ScrollIntoViewProps } from "Utils/scrollHelpers"
+import { Box, media, Text, THEME_V3 } from "@artsy/palette"
+import { compact } from "lodash"
+import styled from "styled-components"
+import { RouterLink } from "System/Router/RouterLink"
 
-export interface PartnerArtistListProps {
-  artists: PartnerArtistList_artists$data
-  distinguishRepresentedArtists: boolean
-  partnerSlug: string
-  scrollTo: ScrollIntoViewProps
-  displayFullPartnerPage: boolean
-}
+// Values here used to implement column headers in CSS
+const PADDING = 0.5
+const OFFSET =
+  parseInt(THEME_V3.textVariants.sm.lineHeight, 10) + PADDING * 10 * 2
```

Comment:
Honestly we should just design around this. I don't like this hack. It's made even weirder because of Safari. Basically; we calculate an offset value here, then use that to negatively transform the column header (because Safari can't handle negative top margins that break out of a container (!?)), then apply a negative margin to the first element in the column.